packages/node_modules/@webex/internal-plugin-conversation/test/integration/spec/encryption.js

```diff
@@ -185,6 +185,24 @@ describe('plugin-conversation', () => {
           .then((c) => {
             assert.property(c, 'defaultActivityEncryptionKeyUrl');
           }));
+
+        it('sets the conversations\'s defaultActivityEncryptionKeyUrl to null', () => webex.internal.conversation.updateKey(conversation, {uri: null})
+          .then(() => checkov.webex.internal.conversation.get(conversation))
+          .then((c) => {
+            assert.isUndefined(c.defaultActivityEncryptionKeyUrl);
+            assert.property(c, 'kmsResourceObjectUrl');
+          }));
+
+        describe('rotate key', () => {
```

Comment:
Should this be something like:
```suggestion
        describe('when the key has been set to null', () => {
```